File: src/main/java/net/dontdrinkandroot/gitki/wicket/headeritem/StyleCssHeaderItem.java
package net.dontdrinkandroot.gitki.wicket.headeritem;

import org.apache.wicket.markup.head.CssReferenceHeaderItem;
import org.apache.wicket.request.Url;
import org.apache.wicket.request.resource.UrlResourceReference;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class StyleCssHeaderItem extends CssReferenceHeaderItem
{
    public StyleCssHeaderItem()
    {
        super(
                new UrlResourceReference(Url.parse("css/style.css")).setContextRelative(true),
                null,
                null,
                null
        );
    }
}

File: src/main/kotlin/net/dontdrinkandroot/gitki/wicket/headeritem/StyleCssHeaderItem.kt
package net.dontdrinkandroot.gitki.wicket.headeritem

import org.apache.wicket.markup.head.CssReferenceHeaderItem
import org.apache.wicket.request.Url
import org.apache.wicket.request.resource.UrlResourceReference

class StyleCssHeaderItem : CssReferenceHeaderItem(
    UrlResourceReference(Url.parse("css/style.css")).setContextRelative(true),
    null,
    null,
    null
)